db/controllers/controllers.js

```diff
@@ -34,12 +34,29 @@ exports.getAllArticles = (req, res, next) => {
   };
 
 
-
-  exports.getArticleComments = (req, res, next) => {
+  exports.getComments = (req, res, next) => {
     const { article_id } = req.params;
     selectArticleComments(article_id)
-    .then((comments) => {
-			res.status(200).send({ comments });
-		})
-		.catch(next);
+
+      .then((comments) => {
+        res.status(200).send({ comments });
+      })
+      .catch(next);
   };
+  
+
+exports.postComment = (req, res, next) => {
+
+  const body = req.body
+	const article_id = req.params.article_id
+	
+  checkArticleId(article_id)
```

Comment:
We don't need to do a separate check here, when posting if the article id is not found it will violate the foreign key restraint so psql will throw an error for us.